### DISCUSSION TITLE
Cannot there be duplicate entity names of multiple connections in mikroorm?

### DISCUSSION THREAD

**Discussion by Lydanne** [mikro-orm/mikro-orm#2607] [Q&A]
Cannot there be duplicate entity names of multiple connections in mikroorm?

**Comment by B4nan** [comment]
it should be possible nowadays

**Reply by Lydanne** [reply]
Are there any documents?

**Reply by B4nan** [reply]
Nope, there is nothing special needed to achieve this.

Apparently you have a problem, so you will need to share details. If you see some validation message from the ORM, it definitely discovered entity with the same name twice, and that is not allowed in a single ORM instance.

**Reply by Lydanne** [reply]
I don't seem to be ideal in TS mode
